projects/packages/forms/src/util/adjust-dashboard-height.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * Adjusts the height of the dashboard container to the available height, including WP notices.
+ *
+ * @param container - The container element to adjust the height of.
+ */
+export const adjustDashboardHeight = ( container: HTMLElement ) => {
+	const applyDynamicHeight = () => {
+		const rect = container.getBoundingClientRect();
+		const availableHeight = Math.max( window.innerHeight - rect.top, 0 );
+
+		container.style.height = `${ availableHeight }px`;
+	};
+
+	requestAnimationFrame( () => {
+		applyDynamicHeight();
+	} );
+
+	const resizeHandler = () => applyDynamicHeight();
+	window.addEventListener( 'resize', resizeHandler );
```

Comment:
Should we also remove this resize handler once it is not needed anymore?

Reply:
My worry is that we might be leaving this resize observer around when we load this in ciab.